--- qsa-x11-free-beta3/src/qsa/qsa.pro.orig	2003-04-15 20:13:17.000000000 +0200
+++ qsa-x11-free-beta3/src/qsa/qsa.pro	2003-04-15 20:13:28.000000000 +0200
@@ -31,8 +31,6 @@
 # win32-msvc.net:QMAKE_LFLAGS += /INCREMENTAL:no /FIXED:no
 
 # I think this belongs here
-isEmpty(QT_SOURCE_TREE):QT_SOURCE_TREE=$(QTDIR)
-isEmpty(QT_INSTALL_PREFIX):QT_INSTALL_PREFIX=$(QTDIR)
 
 INCLUDEPATH	+= ../engine \
 		   $$QT_SOURCE_TREE/tools/designer/interfaces \

