--- qsa-x11-free-1.1.0/src/qsa/qsa.pro.x	2004-01-30 13:28:06.000000000 +0100
+++ qsa-x11-free-1.1.0/src/qsa/qsa.pro	2004-12-13 17:51:40.542572872 +0100
@@ -4,11 +4,7 @@
 TARGET 		= qsa
 VERSION  	= 1.1.0
 
-isEmpty(QSA_INSTALL_PREFIX) {
-  QSA_BUILD_PREFIX=$$(QTDIR)
-} else {
-  QSA_BUILD_PREFIX=../..
-}
+QSA_BUILD_PREFIX=../..
 
 DESTDIR         = $$QSA_BUILD_PREFIX/lib
 DLLDESTDIR      = $$QSA_BUILD_PREFIX/bin
@@ -42,8 +38,6 @@
 }
 
 # I think this belongs here
-isEmpty(QT_SOURCE_TREE):QT_SOURCE_TREE=$(QTDIR)
-isEmpty(QT_INSTALL_PREFIX):QT_INSTALL_PREFIX=$(QTDIR)
 
 INCLUDEPATH	+= ../engine \
 		   ../../tools/designer/editor \
--- qsa-x11-free-1.1.0/src/plugin/plugin.pro.orig	2004-01-30 13:28:06.000000000 +0100
+++ qsa-x11-free-1.1.0/src/plugin/plugin.pro	2004-12-13 18:15:52.786886896 +0100
@@ -1,8 +1,4 @@
-isEmpty(QSA_INSTALL_PREFIX) {
-  QSA_BUILD_PREFIX=$$(QTDIR)
-} else {
-  QSA_BUILD_PREFIX=../..
-}
+QSA_BUILD_PREFIX=../..
 SOURCES        += qseditorplugin.cpp
 DESTDIR	        = $$QSA_BUILD_PREFIX/plugins/designer
 TARGET          = qseditorplugin
